mir-module/src/main/resources/META-INF/resources/js/mir/xeditor-form.js

```diff
@@ -1,5 +1,27 @@
 $(document).ready(function() {
 
+    select2()
+      .then(() => console.info("select2 applied"))
+      .then(() => $('.mir-select-searchable').on('select2:open', function (e) {
+            $(".select2-container--default .select2-results > .select2-results__options")[0].style = "max-height:400px";
+        })
+      )
+      .catch(Error => {
+          console.warn("select2 is not available")
+      });
+
+    /**
+     * Initializes select2 on the webpage for elements with class attribute 'mir-select-searchable'.
+     * */
+    async function select2() {
+        if (typeof jQuery.fn.select2 === "function") {
```

Comment:
JQuery...

Reply:
select2 requires jQuery. if this is a no-go we must look for another framework

Reply:
or use `datalabels`, as for me I do not need select2 for selecting ROR